pkgs/tools/security/pass/extensions/audit/default.nix

```diff
@@ -34,7 +26,7 @@ pythonPackages.buildPythonApplication rec {
 
   postPatch = ''
     substituteInPlace audit.bash \
-      --replace-fail 'python3' "${pythonEnv.interpreter}"
+      --replace python3 "${lib.getExe python3}"
```

Comment:
this should be `replace-fail`

Reply:
eh, `--replace` is so deep in my muscle memory, it will take me a while to get rid of that ;-)

Thanks!